doc/source/intro_rnaquantification.rst

```diff
@@ -0,0 +1,83 @@
+
+Additional Considerations:
+An RNASeq experiment result is the output of running an analysis pipeline on a set of read data.  The result should have metadata that describes the pipeline that was used in detail.  It should include the identity of the input reads in enough detail to retrieve the read data.  All software used should include version, parameters and command line.  Any genome or transcriptome annotations used should be described in enough detail to retrieve the exact version used.  Any changes in software, parameters, annotations or other analysis details should result in a new RNASeq experiment result associated with that input read data.
+
+Case 2: Obtain quantification data for one or more features (genes) for comparison between  multiple RNASeq experiments
+
+User desires:
+Feature quantification data (numeric) for one or more features identified in one or more RNASeq experiment results.  User will provide a list of one or more features for which results should be returned.  If a feature list is not provided, all quantification results for the selected RNASeq experiment(s) should be returned.  User will provide a list of one or more RNASeq experiments to obtain quantification results from.
+
+Additional Considerations:
+In order to request quantifications for comparison with either repository datasets or a local dataset the user needs to determine if the repository RNASeq experiment is comparable to other datasets.  Pipeline metadata needs to be provided so that this can be determined:
+Sample-level: bio sample data such as tissue type, collection methods, sample preparation protocol, library generation protocol, spikes used
+Read-level: sequencer type and protocol, sequence data generation software version, parameters and command line
+Quantification-level: genome annotation, transcriptome annotation if any, software pipeline including versions, parameters and command line
+Batch-level: Adjustments or normalization done at the batch level
+
+Case 3: Obtain input data to use in Assembly activities
+
+User desires:
+Sequence level read data for both mapped and unmapped reads in the associated RNA experiment.  For typical read data this is contained in the fastq file(s) produced by the sequencer pipeline.  The API should either provide the original fastq or the read data necessary and sufficient to generate it.  It is desirable to be able to easily retrieve all the related reads at the fragment level for downstream analysis.  At this time, these would be either single or paired reads but for future-proofing the API should be able to handle the delivery of an arbitrary number of reads for a specific fragment.
+
```

Comment:
the current GA4GH reads API is intended to support unaligned reads.

Maciek Smuga-Otto <notifications@github.com> writes:
> In doc/source/intro_rnaquantification.rst:
> 
> > +
> > +User desires:
> > +Feature quantification data (numeric) for one or more features identified in one or more RNASeq experiment results.  User will provide a list of one or more features for which results should be returned.  If a feature list is not provided, all quantification results for the selected RNASeq experiment(s) should be returned.  User will provide a list of one or more RNASeq experiments to obtain quantification results from.
> > +
> > +Additional Considerations:
> > +In order to request quantifications for comparison with either repository datasets or a local dataset the user needs to determine if the repository RNASeq experiment is comparable to other datasets.  Pipeline metadata needs to be provided so that this can be determined:
> > +Sample-level: bio sample data such as tissue type, collection methods, sample preparation protocol, library generation protocol, spikes used
> > +Read-level: sequencer type and protocol, sequence data generation software version, parameters and command line
> > +Quantification-level: genome annotation, transcriptome annotation if any, software pipeline including versions, parameters and command line
> > +Batch-level: Adjustments or normalization done at the batch level
> > +
> > +Case 3: Obtain input data to use in Assembly activities
> > +
> > +User desires:
> > +Sequence level read data for both mapped and unmapped reads in the associated RNA experiment.  For typical read data this is contained in the fastq file(s) produced by the sequencer pipeline.  The API should either provide the original fastq or the read data necessary and sufficient to generate it.  It is desirable to be able to easily retrieve all the related reads at the fragment level for downstream analysis.  At this time, these would be either single or paired reads but for future-proofing the API should be able to handle the delivery of an arbitrary number of reads for a specific fragment.
> > +
> 
> These reads are from an earlier stage of processing than the reads in a SAM/BAM
> file - would our GA4GH reads API be able to represent these successfully?
> 
> —
> Reply to this email directly or view it on GitHub.*
>